repo-scripts/size-analysis/cli.ts

```diff
@@ -89,4 +89,4 @@ yargs
     argv => runBundleAnalysis(argv as any)
   )
   .help()
-  .parseSync();
+  .parse();
```

Comment:
I think this is going to cause an unhandled promise error from Node. Why do you change it?

Reply:
Reverted back.